app/(marketing)/layout.tsx

```diff
@@ -5,7 +5,7 @@ import MarketingNavbar from "./MarketingNavbar";
 const MarketingLayout = ({ children }: { children: React.ReactNode }) => (
   <div className="flex flex-col min-h-screen">
     <MarketingNavbar />
-    <main className="container mx-auto p-4 md:px-8 flex-grow">{children}</main>
```

Comment:
I think the container should probs be here so that the stuff doesn't go to the very edge, it looks a little cramped now (unless it looks worse when you leave it in)

Reply:
Lowkey it looks weird when you leave it in cuz then there's weird borders since i have background changes each section

Reply:
leave the `flex-grow p-4 md:px-8` and put `container mx-auto` inside of the primary container of each section (same place you declare the background)